firebase-firestore/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
   Bundles contain pre-packaged data produced with the NodeJS Server SDK and
   can be used to populate Firestore's cache without reading documents from
   the backend.
+- [fixed] Fix a Firestore bug where local cache inconsistencies were
+  unnecessarily being resolved, resulting in the incompletion of `Task` objects
```

Comment:
s/incompletion/something that is easier to parse

Reply:
Done.